scipy/sparse/tests/test_base.py

```diff
@@ -756,23 +756,39 @@ def test_sum(self):
         matrices = [dat_1, dat_2, dat_3, dat_4, dat_5]
 
         def check(dtype, j):
+            if j == 1:
+                # At the lowest tested precision for floats and complex
+                # numbers, the sparse matrix sum method can lead to different
+                # results in the lower decimals from the dense matrix sum
+                # method. This necessitaties a more lenient comparison between
+                # the resulting matrices.
+                if dtype == np.complex64:
+                    decimal = 4
+                else:
+                    decimal = 5
```

Comment:
That seems okay to me. I realize now that my suggestion doesn't make sense for the `j == 1` case, because the matrix will be entirely dense anyway.